lizmap/tooltip.py

```diff
@@ -51,6 +52,16 @@ def create_popup_node_item_from_form(
         regex = re.compile(r"[^a-zA-Z0-9_]", re.IGNORECASE)
         a = ''
         h = ''
+
+        if isinstance(node, QgsAttributeEditorElement):
+            # for text widgets
+            if node.type() == 6:
```

Comment:
```suggestion
            # TODO QGIS_VERSION_INT 3.32 change to "Qgis.AttributeEditorType.TextElement"
            if node.type() == 6:
```

I don't like integers comparaison because this is fragile and we need to check in the source code to understand.
I see the API as changed so let's add a comment to upgrade this integer comparaison later ASAP.

Reply:
me neither :) but thanks for the comment, we just have to wait for 3.32 to have a more readable check